oci/layout/oci_dest.go

```diff
@@ -81,6 +83,9 @@ func (d *ociImageDestination) SupportsSignatures(ctx context.Context) error {
 }
 
 func (d *ociImageDestination) DesiredLayerCompression() types.LayerCompression {
+	if d.uncompress {
+		return types.PreserveOriginal
```

Comment:
`types.PreserveOriginal` is _not_ the same thing as “decompress” (that would be `types.Decompress`). (And if I understand the situation correctly, you’d prefer “preserve original”, because that allows duplicate detection to work in more cases.)

So, name the boolean maybe `OCIAcceptUncompressedLayers`? `OCIDoNotForceLayerCompression`? And the private boolean accordingly.

Reply:
@mtrmac updated with `OCIAcceptUncompressedLayers` name.